templates/default/disable_ipv6_loopback.sh.erb

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Disable the ipv6 loopback address to make chef server and EMPD happy
+if ! grep net.ipv6.conf.lo.disable_ipv6 /etc/sysctl.conf; then
+  echo "net.ipv6.conf.lo.disable_ipv6=0" | sudo tee -a /etc/sysctl.conf
```

Comment:
@ryancragun Right, apologies for tangent, my comment was simply to say that the name "disable_ipv6_loopback.sh.erb" is a bit of misnomer since it won't disable ipv6 for the lo.